otx/cli/tools/train.py

```diff
@@ -222,11 +248,215 @@ def main():
     assert resultset.performance is not None
     print(resultset.performance)
 
-    if args.save_logs_to:
-        tmp_path = task.project_path
-        logs_path = os.path.join(args.save_logs_to, tmp_path.split("/")[-1])
-        shutil.copytree(tmp_path, logs_path)
-        print(f"Save logs: {logs_path}")
+    if args.gpus:
+        multigpu_manager.finalize()
+
+
+class MultiGPUManager:
```

Comment:
I'm not sure this is right place for MultiGPUManager.
I prefer short codes for cli

Reply:
but it's only used in train.py. Do you think that it's better to write this class in separate file?

Reply:
I'm not sure, but otx/cli/utils looks fine